src/functions/usdallchannelscount.md

```diff
@@ -11,13 +11,14 @@ This function returns the amount of channels the bot has access to globally. You
 ```php
 $allChannelsCount[type?]
 ```
-### Fields
+### Property 
```

Comment:
this doesn't match the contributing-example guidelines

Reply:
I also have updated contribution guidelines, since properties makes more sense. (It would be better to use parameters but properties would be better since users understand it as like that.)